src/app/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -21,6 +22,7 @@ const Canonical = () => {
   const onCookieAccept = () => {
     cookieOnAllow();
     updateCookiePolicy();
+    onDismissFocusRef?.current?.querySelector('a')?.focus();
```

Comment:
As you rightly raised, we probably also want to do this when a user declines cookie consent.

Reply:
When you reject consent on canonical, it navigates to a page outside of the Simorgh stack: https://www.bbc.co.uk/usingthebbc/your-data-matters 
There is no need for focus logic in practice as we just navigate to another page but I guess it is harmless? Maybe this behaviour could change in the future and this logic could become useful.

Reply:
Ah, in this case it is unnecessary to call `focus()` on reject right now, @HarryVerhoef. Thanks, @andrewscfc.